src/Bicep.Core.IntegrationTests/Semantics/SemanticModelTests.cs

```diff
@@ -28,7 +28,8 @@ public class SemanticModelTests
         [NotNull]
         public TestContext? TestContext { get; set; }
 
-        [DataTestMethod]
+        // TODO:  handle varying linter diagnostic expectations for data driven test
+        [DataTestMethod, Ignore]
```

Comment:
>Ignore [](http://example.com/codeflow?start=25&length=6)

[MVP] Does it cause this test to be skipped? If yes, this needs to be removed before merging.

Reply:
A baseline update should take care of this test btw.

Reply:
Ignored tests are now fixed and running.